updatecli/updatecli.d/ruby.yml

```diff
@@ -0,0 +1,87 @@
+---
+name: Bump Ruby version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  # use dockerimage last version while no solution for https://github.com/updatecli/updatecli/issues/1951
+  # rubyLastestRelease:
+  #   kind: githubRelease
+  #   name: "Get the latest Ruby version"
+  #   spec:
+  #     owner: "ruby"
+  #     repository: "ruby"
+  #     token: "{{ requiredEnv .github.token }}"
+  #     versionFilter:
+  #       kind: latest
+  #   transformers:
+  #     - trimprefix: "v"
+  #     - replacers:
+  #       - from: "_"
+  #         to: "."
+  rubyDockerImageLatestVersion:
+    kind: dockerimage
+    name: "Get the latest Ruby version"
+    spec:
+      image: "ruby"
+      tagFilter: >-
+        \d\.\d\.\d$
+      versionfilter:
+        kind: regex
+        pattern: >-
+          \d\.\d\.\d$
+
+conditions:
+  checkForChocolateyPackage:
+    kind: shell
+    disablesourceinput: true # Do not pass source as argument to the command line
+    spec:
+      command: curl --silent --show-error --location --fail --output /dev/null https://community.chocolatey.org/packages/ruby/{{ source "rubyDockerImageLatestVersion" }}.1
```

Comment:
Why is there a `.1` at the end? That's not clear when reading the manifest.

Reply:
that coming from chocolatey https://community.chocolatey.org/packages/ruby/#versionhistory